test/endtoend/operator/101_initial_cluster.yaml

```diff
@@ -8,12 +8,12 @@ metadata:
   name: example
 spec:
   images:
-    vtctld: vitess/lite:v12.0.3
-    vtgate: vitess/lite:v12.0.3
-    vttablet: vitess/lite:v12.0.3
-    vtbackup: vitess/lite:v12.0.3
+    vtctld: vitess/lite:v13.0.0
```

Comment:
should this be 13.0.0 or a later minor version?

Reply:
We can do any/all. I just added this to check that we are able to upgrade from the earliest release